editor/src/templates/vscode-editor-outer-iframe/index.html

```diff
@@ -9,5 +9,13 @@
       id="vscode-outer-root"
       style="width: 100%; height: 100%; display: flex; flex-direction: column;"
     ></div>
+
+    <% if (VITE) { %>
```

Comment:
Only while hosted from Vite?

Reply:
Yeah. Vite works by starting from the various `index.html` files it finds in the specified entry point folder, and resolves the source files from there, meaning we need to explicitly include the scripts in the HTML. Webpack works in the opposite direction, starting with the source files, and then injecting the output modules into the HTML via a plugin.